### DISCUSSION TITLE
I want to try to read the document in another path

### DISCUSSION THREAD

**Discussion by Chengyunlai** [nuxt/content#2090] [Q&A]
Does the search path for the document have to be the content directory under the Nuxt project?

In the code, I want to replace `me`, because it's in the content directory, and I don't want to do that.
```
<template>
    <ContentRenderer :value="result"></ContentRenderer>
</template>

<script setup>
  const result = ref(null); // 使用ref创建一个响应式引用

  onMounted(async () => {
    const contentQuery = queryContent('me').findOne();
    result.value = await contentQuery; // 使用await获取结果并将其赋值给result.value
  });
</script>
```

**Comment by nobkd** [comment]
Not completely sure what you mean, but probably you're looking for this option: https://content.nuxtjs.org/api/configuration#sources

This might also be worth a look, as it's about the same thing: #2107
